Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -807,8 +834,11 @@ def _exec_editor_test(self, request: _pytest.fixtures.FixtureRequest,
             return_code = editor.get_returncode()
             editor_log_content = editor_utils.retrieve_editor_log_content(run_id, log_name, workspace)
             # Save the editor log
-            workspace.artifact_manager.save_artifact(os.path.join(editor_utils.retrieve_log_path(run_id, workspace), log_name),
-                                                     f'({run_id}){log_name}')
+            path_to_artifact = os.path.join(editor_utils.retrieve_log_path(run_id, workspace), log_name)
+            full_log_name = f'({run_id}){log_name}'
+            dest_path = workspace.artifact_manager.save_artifact(path_to_artifact, full_log_name)
+            _split_batched_editor_log_file(dest_path, full_log_name)
```

Comment:
I think _exec_editor_test() is not used for batched tests and therefore does not need to be split.